lib/views/teach_word_view.dart

```diff
@@ -207,9 +238,13 @@ class _TeachWordViewState extends State<TeachWordView>
                                       icon: const Icon(Icons.play_arrow),
                                       selectedIcon: const Icon(Icons.pause),
                                       onPressed: !controller.isQuizzing
-                                          ? () {
+                                          ? () async {
```

Comment:
Align to the 2-space indent style with previous codes. Remove additional indents from 4 to 2.